### ISSUE TITLE
Lighthouse audit always return warning 'Missing source maps for large first-party JavaScript'

### ISSUE THREAD

**Issue by danurahadi** [GoogleChrome/lighthouse#14967]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

https://sportz.picassorewards.com:8443/login

### What happened?

I face an issue of Chrome Lighthouse Audit. 

I build a web app that loads Sock JS minified file.

The file is -> [https://unpkg.com/sockjs-client@1.5.0/dist/sockjs.min.js][1]

As you can see, there is **//# sourceMappingURL** at the end of the file.

Also, there is the source map file -> [https://unpkg.com/sockjs-client@1.5.0/dist/sockjs.min.js.map][2]

  [1]: https://unpkg.com/sockjs-client@1.5.0/dist/sockjs.min.js
  [2]: https://unpkg.com/sockjs-client@1.5.0/dist/sockjs.min.js.map

### What did you expect?

I think that warning should not appear.

### What have you tried?

I have tried to load the SockJS minified file from the CDN and do the Lighthouse Audit several times, but it's still produce the same warning :

**Missing source maps for large first-party JavaScript**

Environment :

 - Ubuntu 22.04
 - Brave Browser v1.50.114

Am I missing something here or there is a bug in the Lighthouse?

Thank you.

### How were you running Lighthouse?

Chrome DevTools

### Lighthouse Version

10.0.1

### Chrome Version

112.0.5615.49

### OS

Linux Ubuntu 22.04

**Comment by adamraine** [comment]
I don't think sockjs.min.js is causing the audit to fail, it's the other script (https://sportz.picassorewards.com:8443/_nuxt/entry.86f859b9.js) which doesn't have a source map. Are you seeing that as well?

I did also notice another error while testing your page though:

![Screenshot 2023-04-14 at 2 36 16 PM](https://user-images.githubusercontent.com/6752989/232159152-ea4e48b6-a17c-431f-affc-4e303550acc6.png)

@alexnj can you investigate?

**Comment by danurahadi** [comment]
@adamraine I'm not sure about the script you mention because it's generated by Nuxt.js.
And also that error message (screenshot) didn't show up on my local machine.

Maybe you Chrome guys can see the audit results from my local machine below :

![lighthouse-audit](https://user-images.githubusercontent.com/33719806/232737488-afc91330-b415-455f-8f19-b23392477a0a.png)

Thanks so much for your response.

**Comment by adamraine** [comment]
Yeah, looks like the audit is failing because `entry.c1164377.js` doesn't have a source map, and not because of `sockjs.min.js`. This file had a different name on my.

**Comment by alexnj** [comment]
I created a follow up issue #14996 to track the chrome plugin URL breaking the audit case.

**Comment by alexnj** [comment]
Re: the original issue — it seems like the inclusion of passing source-map entries with a partially failing audit `valid-sourcemaps` result is the cause for confusion. We could either drop the passed entries, or rearrange them under a sub-headings as `Passed` and `Failed` to resolve the ambiguity in the UI. Wdyt @adamraine ?

**Comment by adamraine** [comment]
Not sure about using `Passed` / `Failed` as headings since we need the sub-headings to display the failed messages. There are definitely a few ways to approach this though:

- `Passed` / `Failed` indicators in a table column
- Put the passed / failed scripts in entirely separate tables
- Add a diagnostic message as a sub-item to passed scripts

**Comment by danurahadi** [comment]
@adamraine @alexnj thank you all guys, for the response.
btw, what's the meaning of P2 label?

**Comment by adamraine** [comment]
It's the issue priority on a 1-3 scale

**Comment by danurahadi** [comment]
ah i see. thank you.
